src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -784,7 +784,7 @@ namespace InternalHeatGains {
                                 {
                                     auto const clothingType(AlphaName(10));
```

Comment:
Are you sure this doesn’t result in a string copy? Maybe add &? Or better yet std::string_view?

Reply:
:+1: , made it a string_view.